lib/rspec/support/source/node.rb

```diff
@@ -45,13 +45,18 @@ def location
           @location ||= args.find { |arg| arg.is_a?(Location) }
         end
 
-        def each(&block)
+        def each
           return to_enum(__method__) unless block_given?
 
-          yield self
+          outstanding_nodes = []
+          outstanding_nodes << self
```

Comment:
Can these two lines be combined into `outstanding_nodes = [self]`?

Reply:
Looks like combining these lines causes a segfault on 1.9.2.

Reply:
Bizarre! Let’s keep them separate then.